### PR TITLE
Mustafa

### PR DESCRIPTION
**Pull request by mustafa3342**

### CODE REVIEW

**Review by ozkankrm — COMMENTED** [review]
good job mate.
you will never give up

**Review by ozkankrm — COMMENTED** [review]
perfect

**Review by ozkankrm — COMMENTED** [review]
ggg

**Comment by ozkankrm** [comment]
ı closed the pull request

**Review by mustafa3342 — COMMENTED** [review]
ok

File: src/test/java/Class4.java [hunk 1]
```diff
@@ -3,6 +3,8 @@ public static void main(String[] args) {
         System.out.println("Hello");
         //Add class4
         // who did add this line
+        
+        //why was you added this line
     }
```
Comment (mustafa3342) [inline src/test/java/Class4.java:9]: problem is solved

**Review by mustafa3342 — COMMENTED** [review]
look at

**Comment by mustafa3342** [comment]
why merging is blocked

**Review by ozkankrm — COMMENTED** [review]
great

**Comment by ozkankrm** [comment]
ı reviewed and added comment.

**Review by ozkankrm — COMMENTED** [review]
changed

**Comment by ozkankrm** [comment]
changed updated

**Review by ozkankrm — APPROVED** [review]
approve

**Comment by mustafa3342** [comment]
hallettik

**Pull request merged**